projects/almalinux-8/Dockerfile

```diff
@@ -52,6 +52,9 @@ RUN rpm --import https://packages.irods.org/irods-signing-key.asc && \
 RUN rpm --import https://core-dev.irods.org/irods-core-dev-signing-key.asc && \
     wget -qO - https://core-dev.irods.org/renci-irods-core-dev.yum.repo | tee /etc/yum.repos.d/renci-irods-core-dev.yum.repo
 
+RUN yum install -y dnf-plugins-core
+RUN yum config-manager --set-enabled powertools
```

Comment:
Would it be possible to add these to the build step which begins on line 3 to avoid adding the two additional layers?

Reply:
Or, if you could combine these two `RUN` commands into one that would be fine, too.

Reply:
@d-w-moore *poke*

Reply:
Done